build/configs/artik053/tc/defconfig

```diff
@@ -893,6 +893,7 @@ CONFIG_SMARTFS_ALIGNED_ACCESS=y
 # CONFIG_SMARTFS_JOURNALING is not set
 # CONFIG_SMARTFS_SECTOR_RECOVERY is not set
 CONFIG_FS_PROCFS=y
+CONFIG_FS_AUTOMOUNT_PROCFS=y
```

Comment:
Please copy and paste configs in "SMARTFS options" only made by menuconfig automatically.